unix/xserver/hw/vnc/vncInput.c

```diff
@@ -207,6 +207,24 @@ static int vncPointerProc(DeviceIntPtr pDevice, int onoff)
 		btn_labels[5] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_HWHEEL_LEFT);
 		btn_labels[6] = XIGetKnownProperty(BTN_LABEL_PROP_BTN_HWHEEL_RIGHT);
 
+    /*
```

Comment:
Indentation is off here.

Reply:
Whoops, looks like the file uses tabs and not spaces.